deploy-board/deploy_board/templates/deploys/deploy_history.tmpl

```diff
@@ -63,6 +64,7 @@
                 {% endif %}
             </td>
             <td>{{ deploy_summary.deploy.operator }}</td>
+            <td><a href="/clouds/accounts/{{ deploy_summary.account.provider }}/{{ deploy_summary.account.cell }}/{{ deploy_summary.account.id }}">{{ deploy_summary.account.id }}</a></td>
```

Comment:
Account id is UUID which represents our internal account id and don't make sense for our users. It's better to display AWS account number and account name here how it was done for environment landing page in the PR https://github.com/pinterest/teletraan/pull/1522

Let's change link to make it looks like:
![Screenshot 2024-03-18 at 10 25 50 AM](https://github.com/pinterest/teletraan/assets/20383875/fb197cdc-39b5-49a4-a546-683329c6a6c3)

https://github.com/pinterest/teletraan/pull/1522/files#diff-7929c2a17902ed9d0b824bf889f4e9b06096b050f0d11e137ba1f7a957132fbcR81

Reply:
Do you mean this code https://github.com/pinterest/teletraan/pull/1522/files#diff-33d8c883629bab6f1a7dee3462fa5794c1ff357ee6f9a536b1ff169ac45181beR283-R297 ?

If so, I think we can move the function `create_legacy_ui_account` to the separated file and make it library function, next reuse in both places to create ui accounts.

```Python
def create_legacy_ui_account(account_id):
    if account_id == AWS_PRIMARY_ACCOUNT:
        return {
            "name": f"{AWS_PRIMARY_ACCOUNT} / Primary AWS account",
            "ownerId": AWS_PRIMARY_ACCOUNT,
        }
    if account_id == AWS_SUB_ACCOUNT:
        return {
            "name": f"{AWS_SUB_ACCOUNT} / Moka account",
            "ownerId": AWS_SUB_ACCOUNT,
        }
    return {
        "name": f"{account_id} / Sub AWS account",
        "ownerId": account_id,
    }
```